specs/testdata/gcp.yml

```diff
@@ -4,12 +4,13 @@ spec:
   version: v1.0.0
   concurrency: 10
   registry: local
+  path: ~/cq/test
```

Comment:
I believe the `~` symbol will not actually be expanded to your home directory by our CLI, so let's not set the expectation that this should work
```suggestion
  path: /home/cloudquery/cq/test
```